PointOfInteraction/SmartSpot/doc/spec.md

```diff
@@ -0,0 +1,76 @@
+# SmartSpot
+
+## Description
+
+Smart Spots are devices which provide the technology to allow users to interact (suggestions mailbox, co-creation, …) or to obtain extra information (tourism, infotainment, …). The data model contains resources to configure the interaction service such as the broadcasted URL (probably will be a shortened url), the interval between broadcasts (frequency), the availability of the service, transmission power in function of the area to cover, etc.
+
+These resources (except the references to other entities) are mapped from a real device. For that reason, the changes in any of the values of this entity will modify the value in the real device.
+
+## Data Model
+
++ `id` : Unique identifier. 
+
++ `type` : Entity type. It must be equal to `SmartSpot`.
+
++ `physicalURL` : Physical URL broadcasted by the device.
```

Comment:
what's the difference between this physicalURL and the announcedURL of a SmartPointOfInteraction?